src/components/PublicationForm.js

```diff
@@ -10,8 +10,8 @@ export default function PublicationForm(){
     const [postLink, setPostLink] = useState("");
     const [content, setContent] = useState(""); 
     const [actionDisabled, setActionDisabled] = useState(false);
-        
-    const navigate = useNavigate()
+    let navigate = useNavigate();    
```

Comment:
why change this to let instead of const?

Reply:
I was testing something, but I already removed it.